.bazelci/tests.yml

```diff
@@ -0,0 +1,36 @@
+tasks:
+  hello_macos:
+    name: hello
+    working_directory: integration/hello_bazel
+    platform: macos
+    # I realy want bazel run. Is that allowed?
```

Comment:
use run_targets for bazel run (https://github.com/bazelbuild/continuous-integration/tree/master/buildkite#running-commands-shell-scripts-or-binary-targets)

Reply:
Done.